### ISSUE TITLE
Any change to CT Dynamax Adventures makes them unusable ingame

### ISSUE THREAD

**Issue by resetes12** [kwsch/pkNX#103]
Yesterday I tried editing the Crown Tundra Dynamax legendaries to have all legendaries in both versions. 
When I applied the changes and went ingame, the game worked fine until you save to start the Dynamax Adventure. After saving, the game just stays 2-3 seconds in a black screen and kicks you out of the dialogue with the NPC, resulting in being unable to start the adventure. There's no softlock or anything, you just can't start the adventure.

As that didn't work, I scrapped the old file and I just changed the Mon number 000 (Ivysaur) to be Venasaur, with the rest of the file in its default form. This didn't work, and resulted in the above again.

In summary, **changing anything in the Dynamax Adventures tab makes them completely unusable ingame.**

Using the latest version of pkNX, 1.3.0 update and Pokémon Shield. I can provide more information if needed.

**Comment by kwsch** [comment]
The game has some integrity checks for the data, and even hex editing results in the same timeout.

The editor works; but any sanity checks are undocumented. If someone figures out how the game compares this, then another patcher tool can be made.
